configmap/api-server/src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+quarkus.openshift.expose=true
+quarkus.kubernetes-config.enabled=true
```

Comment:
So this is the main difference between fs vs. api approach?

configmap.yaml files look the same.

Reply:
Yes. Instead of mounting the ConfigMap into the filesystem, we use the `quarkus-kubernetes-config` extension, which creates another `ConfigSource`, which reads the `application.properties` file directly from the Kubernetes API server.